package/metrics_daemon/metrics_daemon/overlay/etc/logrotate.d/metrics_json

```diff
@@ -0,0 +1,7 @@
+/var/log/metrics.json {
```

Comment:
Need to add rotation support to the metrics daemon, in particular we need a line like this:
```
    postrotate
        [ ! -f /var/run/metrics_daemon.pid ] || kill -HUP `cat /var/run/metrics_daemon.pid`
```

Reply:
The `metrics_daemon` uses the `file_append_string` api from `libpiksi/utils` which opens and closes based on the file name. Do you thing that is sufficient for our needs? or is there a race condition that would cause some sort of issue if the rotate happened during the write routine?

Reply:
No, that should be sufficient.